src/Model.jl

```diff
@@ -94,28 +96,43 @@ function Model(lattice::AbstractMatrix{T},
                atoms::Vector{<:Element}=Element[],
                positions::Vector{<:AbstractVector}=Vec3{T}[];
                model_name="custom",
-               n_electrons::Int=sum(n_elec_valence, atoms; init=0),
+               εF=nothing,
+               n_electrons=isnothing(εF) ? n_electrons_from_atoms(atoms) : nothing,
+               # force electrostatics with non-neutral cells; results not guaranteed
+               force_electrostatics=false,
```

Comment:
I had this as `check_electrostatics` because I think it's more suggestive to "disable a check" than to "force" something, don't you think?

Reply:
Well only to people who know the `--force` flags in unix commands, I'd say.

Reply:
force is a pretty universal notion in computing, no? It conveys the sense that you have to be extra careful.